src/types/line.js

```diff
@@ -253,12 +265,29 @@ module.exports = function(Chart) {
 				);
 				ctx.fillStyle = view.labelFontColor;
 				ctx.textAlign = 'center';
-				ctx.textBaseline = 'middle';
-				ctx.fillText(
-					view.labelContent,
-					view.labelX + (view.labelWidth / 2),
-					view.labelY + (view.labelHeight / 2)
-				);
+
+				if (view.labelContent !== null && view.labelContent.indexOf("\n") !== -1) {
```

Comment:
do we need this `if/else` check? `view.labelContent.split("\n")` will return an array with a single element and then it should be able to be processed just like if there were multiple elements